style.css

```diff
@@ -1,624 +1,995 @@
+  width: calc(100% + 16px);
+  height: 40px;
+  text-align: center;
+  text-decoration-line: none;
+  padding: 10px;
+  color: #fff;
+}
+
+.recent-work-pic {
+  width: 100%;
+  object-fit: contain;
+}
+
+.recent-work-pic img {
+  width: 100%;
+  height: auto;
+  object-fit: fill;
+  display: block;
+}
+
+.recent-work-details {
+  width: 100%;
+  margin: 0;
+}
+
+#work-third {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/sec-image.png");
+}
+
+#fourth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/third-image.png");
+}
+
+#fifth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/first-image.png");
+}
+
+#sixth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/sec-image.png");
+}
+
+#seventh-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/third-image.png");
+}
+
+.work-title {
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  margin: 5px 0 5px 0;
+}
+
+.works-description {
+  margin: 5px 0 5px 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #091e42;
+}
+
+#personal-data {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  max-width: 375px;
+  height: 1900px;
+  padding: 20px 20px;
+  background-color: #000;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+#my-cv-here {
+  background-image: url("./images/about_me_image.svg");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  height: 650px;
+  object-fit: contain;
+}
+
+.card-link {
+  text-decoration-line: none;
+  color: #fff;
+  background-color: #f55800;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  padding: 12px;
+  gap: 10px;
+  width: 171px;
+  height: 48px;
+}
+
+#my-cv-here a:active {
+  background: #e05304;
+}
+
+#my-cv-here a:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+#my-cv-here a:disabled {
+  background: #f4f1f0;
+  border: none;
+  color: #5b5957;
+}
+
+#my-cv-here a:enabled {
+  background: #f55800;
+  border: 1px solid #f55800;
+  color: #fff;
+}
+
+#my-cv-here h2 {
+  color: #f55800;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 82px;
+}
+
+#my-cv-here p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+#my-skills {
+  border-top: 1px solid #888;
+  clear: both;
+  text-align: center;
+  padding: 120px 0 0 0;
+}
+
+#my-skills ul {
+  font-size: 35px;
+  float: center;
+  text-indent: -30px;
+}
+
```

Comment:
- [ ] Please try to adjust your styles on the following elements in order to match the following design. Please look at the following comparison 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/200007699-5235e51a-84cc-4155-af3d-39330a20f445.png)
**Given design**
![image](https://user-images.githubusercontent.com/90222110/200007815-8d1bf242-2f79-4465-8582-c0b656a28632.png)

To be specific, you can add `text-weight: bold` 😉

Reply:
![skill-set](https://user-images.githubusercontent.com/32777408/200087451-6e177fdc-9f61-43c5-9505-f39bf53cb554.png)

This issue has been resolved. @BertrandConxy 

I have attached an image to this so you can see it with easy